sema4ai/vscode-client/src/pythonExtIntegration.ts

```diff
@@ -144,6 +157,29 @@ export async function installWorkspaceWatcher(context: ExtensionContext) {
                             }
                         });
                 }
+
+                const actionPackages = await listActionPackages();
+                const actionPackageRoot = matchActionPackageToFile(actionPackages, docURI);
+                if (actionPackageRoot) {
+                    const metadataFilePath = await findMetadataFilePath(actionPackageRoot);
+
+                    /* If the Python file changed, and the metadata.json is present, ask the user if they want to update or delete it. */
+                    if ((docURI.fsPath.endsWith("package.yaml") || isPythonFile(docURI.fsPath)) && metadataFilePath) {
```

Comment:
I think it would be nice to add `isPackageYamlFile` helper, similar to `isPythonFile` or `isEnvironmentFile`